.travis-tox.ini

```diff
@@ -36,6 +36,7 @@ envlist =
     api
     {py36,py37,py38,pypy3}-cover
     {py36,py37,py38,pypy3}-nocov
+    {py36,py37,py38}-mypy
```

Comment:
This seems to work, but should this line by just "mypy" (single environment)?

It looks like you just define a single "mypy" target below ``[testenv:mypy]`` (although the Tox syntax is not fresh in my mind) and moreover call it via TravisCI along side the "sdist" and "bdist_wheel" targets using ``TOXENV=sdist,bdist_wheel,mypy``